webapp/src/components/gitlab_issue_selector.tsx

```diff
@@ -73,95 +72,77 @@ export default class GitlabIssueSelector extends PureComponent<PropTypes, StateT
             return issues.map((issue) => {
                 const projectParts = issue.web_url.split('/');
                 let prefix = '';
+                //Extract "username/projectName" from the issueURL parts
```

Comment:
```suggestion
                // Extract "username/projectName" from the issueURL parts
```
I think the comment can be more descriptive or the code can be more readable. We can discuss this.

Reply:
discussed